test/resources/hosted_mode/managed-cluster-addon.yaml

```
@@ -4,6 +4,7 @@ metadata:
   name: governance-policy-framework
   annotations:
     addon.open-cluster-management.io/hosting-cluster-name: cluster1
+    addon.open-cluster-management.io/values: '{"global":{"imageOverrides":{"config_policy_controller":"quay.io/stolostron/governance-policy-framework-addon:imagetag"}}}'
```

I think you mean this?
```suggestion
    addon.open-cluster-management.io/values: '{"global":{"imageOverrides":{"governance_policy_framework":"quay.io/stolostron/governance-policy-framework-addon:imagetag"}}}'
```

---

good good Thanks!!